app/components/articleSearch/components/autoYearFilter/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { withStyles } from '../../../../helpers/withStylesHelper';
+import ActionTicketManager from '../../../../helpers/actionTicketManager';
+const styles = require('./autoYearFilter.scss');
+
+interface AutoYearFilterProps {
+  query: string | undefined;
+  detectedYear: number | null;
+  handleSetUseAutoYearFilter: (value: React.SetStateAction<boolean>) => void;
+}
+
+const AutoYearFilter: React.FC<AutoYearFilterProps> = ({ query, detectedYear, handleSetUseAutoYearFilter }) => {
+  React.useEffect(
+    () => {
+      if (detectedYear) {
+        ActionTicketManager.trackTicket({
```

Comment:
If you log this information in this block, it can't log the query that has the same detected year.
(ex: machine learning 2016 -> search engine 2016)